Formula/r.rb

```diff
@@ -94,6 +94,20 @@ def post_install
     ln_s site_library, lib/"R/site-library"
   end
 
+  def caveats
+    on_macos do
+      <<~EOS
+        This installation of R does not use binary packages from CRAN.
+        Instead it will install packages from source which will require additional system
+        dependencies and longer install times.
+
+        If you are installing packages from CRAN it is highly recommended that you
+        instead install R with:
+          brew install --cask r
+      EOS
```

Comment:
Let's make this a bit terser:

```suggestion
        CRAN packages will be built from source. If you need binaries, install R with
          brew install --cask r
      EOS
```